src/js/cli.js

```diff
@@ -157,11 +160,15 @@ const fillTemplate = (str, arg) => {
 };
 
 process.on('unhandledRejection', error => {
-  console.error(typeof error === 'string' ? error : error instanceof UsageError ? error.message : error.stack);
+  console.error(
```

Comment:
Clang does some weird stuff...

Reply:
Doesn't seem too weird - it's following the [Rectangle Rule](https://github.com/google/google-java-format/wiki/The-Rectangle-Rule)